src/main/java/hudson/security/LDAPSecurityRealm.java

```diff
@@ -1516,22 +1521,22 @@ public boolean isGeneratingPrefixRoles() {
             return StringUtils.isNotBlank(rolePrefix) || convertToUpperCase;
         }
 
-        public boolean isConvertToUpperCase() {
+        public boolean _isConvertToUpperCase() {
```

Comment:
Why?

Reply:
Oh this was a bit of a mess indeed. See https://github.com/jenkinsci/ldap-plugin/pull/49#discussion_r463740875 amended by https://github.com/jenkinsci/ldap-plugin/pull/49/commits/a9dc4e48687f07fdf4ca374d55cb4ac662f6d425 (which also reduces diff size). Basically the plugin never used the `super` implementation of `convertToUpperCase` and `rolePrefix`; instead https://github.com/jenkinsci/ldap-plugin/blob/9577fe6b8ea685d04fcfe243e750eeea059a88a2/src/main/java/hudson/security/LDAPSecurityRealm.java#L1506-L1515 reimplements something like it, for reasons I do not entirely follow. So to disable the `super` implementations, but still capture values passed to the setters, while accommodating a supertype which newly made the getters (but not setters) `final`, we have these tricks.